packages/faustwp-core/src/components/Toolbar/ToolbarItem.tsx

```diff
@@ -3,19 +3,11 @@ import React, { PropsWithChildren } from 'react';
 type Props = PropsWithChildren<{
   href?: string;
   tabIndex?: number;
-  handleClick?: () => Promise<void>;
 }> & { [key: string]: unknown };
 
-export function ToolbarItem({ href, handleClick, children, ...props }: Props) {
```

Comment:
By removing `handleClick`, will this effect any users that may be adding custom toolbar nodes?

Just want to make sure this isn't a breaking change.

Reply:
Yes, this would affect anyone already using handleClick, as it's now relying on folks to pass in native attributes which should be caught be ...props.

An example of the new ToolbarItem can be seen here, where we are passing in native attributes to separate click events from keyboard events.

I can update the changelog to reflect that it is a breaking change.

Reply:
@josephfusco Yeah I mean toolbar is still "experimental", so maybe no changeset bump, but we should call it out in the  changeset desc.